### ISSUE TITLE
Bug - User not able to enter facilities - discovery

### ISSUE THREAD

**Issue by RakshindaAslam** [department-of-veterans-affairs/va.gov-team#57981]
## Summary
Multiple users have reported the following issues about the selecting/entering facilities on 526ez form:
- Unable to get a drop-down to select the appropriate one
- When they type in the facility name completely, they cannot save it
- Another user gets an error on the evidence page and is not able to move forward "a treatment facility does not meet minimum length of one"

Slack thread - https://dsva.slack.com/archives/C04KW0B46N5/p1683556195129599

## Specs
- Device: 
- Browser: 

## Steps to Reproduce
**Test User:** 

1. Go to: _URL_
2. 

### Actual Result
- Described in the summary above

### Desired Result
User should be able to enter the correct facilities name and move to the next page and submit the form.
- Identify the cause and scope of the issue
- What is considered correct facilities entry?
- Propose a fix 

## Definition of Done
- [ ] Desired result is achieved and/or document decisions made
- [ ] Relevant documentation is updated

---
## How to configure this issue
- [ ] **Labeled with Team** (`vaos`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `design`, `product`)
- [ ] **Labeled with** `Bug`
- [ ] **Assigned to** Product for prioritization

**Comment by navaeddie** [comment]
I was able to reproduce Case# 10144638 (the first issue) on staging.

![Screenshot 2023-05-08 at 10.55.01 AM.png](https://images.zenhubusercontent.com/63dd30048b2159302590655a/b8f085c1-52c4-45d8-a5c6-338e301c2e22)

I'm guessing the screen should autocomplete with options for me to choose from, but no suggestions appear no matter what I press. I wasn't able to advance past this step in the form.

1. What options should be appearing up?
2. How long has this been broken?
3. Running it locally and digging into Network tab might reveal errors related to what facilities should show up.
4. Where is the source of truth on this list of Facilities?

**Comment by RakshindaAslam** [comment]
@kylesoskin @Mottie - would either of you know the answers to the questions in the comment above?

**Comment by Mottie** [comment]
The facilities autocomplete _used_ to be in place, but EVSS shut down that service a few years back. We had plans on switching the autocomplete over to use the new BRD endpoint, but never had the time to do it. So, right now, there isn't any autocomplete functionality for this input - but the code should still be there. I don't think it's making any API calls.

**Comment by christinec-fftc** [comment]
[#58745](Bug Fix - Changes for VAMC field in Supporting Evidence) deployed to prod today related to this issue. The change includes making the VA medical facility name a freeform field so a user should be able to type in a name and not have to use a drop down.

Note that the field is still required so user will still need to fill out a VA medical facility name. I don't see a way to edit this on the review and submit page, so user may need to go back to the Supporting Evidence chapter to fill this out.

**Comment by RakshindaAslam** [comment]
@christinec-fftc - if the field is required for submission, can we make sure that it is filled out  on the supporting evidence page, before moving forward?

**Comment by christinec-fftc** [comment]
@RakshindaAslam Yes, the existing validation is still in place on the supporting evidence > va medical records page.
